.ci/README.md

```diff
@@ -22,20 +22,24 @@ You can use the npm bindings to run tests on all translators:
 (faster, because up to 8 cores are used, such that output will not stay in alphabetical order)
 
 
-## Using the script directly
+## Using the scripts directly
 
 You can use the bash script `checkTranslator.sh` directly to
 run tests on a single translator, e.g.
 ```
 ./checkTranslator.sh ../Amazon.js
 ```
-Moreover, you can skip the warnings by setting the environment
-variable `SKIP_WARN` to `true`:
+Moreover, you can skip the warnings by using the `--skip-warn`
+option:
 ```
-export SKIP_WARN=true
-./checkTranslator.sh ../Amazon.com
+./checkTranslator.sh --skip-warn ../Amazon.com
```

Comment:
So, basically, we just need to test whether path exists as-is, and if it doesn't try `$ROOTDIR/$LABEL.js` (or whatever the variables are).